lib/widgets/gists_item.dart

```diff
@@ -0,0 +1,134 @@
+                  Row(
+                    children: <Widget>[
+                      Avatar(
+                        url: avatarUrl,
+                        size: AvatarSize.small,
+                        linkUrl: '/github/$login',
+                      ),
+                      SizedBox(width: 8),
+                      Expanded(
+                        child: Text.rich(
+                          TextSpan(children: [
+                            TextSpan(
+                              text: '$login / ',
+                              style: TextStyle(
+                                fontSize: 18,
+                                color: theme.palette.primary,
+                              ),
+                            ),
+                            TextSpan(
+                              text: filenames[0].filename,
+                              style: TextStyle(
+                                fontSize: 18,
+                                color: theme.palette.primary,
+                                fontWeight: FontWeight.w600,
+                              ),
+                              // overflow: TextOverflow.ellipsis,
+                            ),
+                          ]),
+                          overflow: TextOverflow.ellipsis,
+                        ),
+                      ),
+                    ],
+                  ),
+                  SizedBox(height: 8),
+                  if (description != null && description.isNotEmpty) ...[
+                    Text(
+                      description,
+                      style: TextStyle(
+                        color: theme.palette.secondaryText,
+                        fontSize: 16,
+                      ),
+                    ),
+                    SizedBox(height: 10),
+                  ],
+                  if (updatedAt != null) ...[
+                    Text(
+                      'Updated ${timeago.format(updatedAt)}',
+                      style: TextStyle(
+                        fontSize: 14,
+                        color: theme.palette.tertiaryText,
+                      ),
+                    ),
+                    SizedBox(height: 10),
+                  ],
+                  DefaultTextStyle(
+                    style: TextStyle(color: theme.palette.text, fontSize: 14),
+                    child: Row(
+                      children: <Widget>[
+                        Container(
+                          width: 12,
+                          height: 12,
+                          decoration: BoxDecoration(
+                            // Error in color: convertColor(langauge ?? languagesColor[language]),
+                            color: convertColor(languagesColor[language]),
+                            shape: BoxShape.circle,
+                          ),
+                        ),
+                        SizedBox(width: 4),
+                        Text(
+                          language,
```

Comment:
This might be null sometimes.

Reply:
I used an `if` statement. Using the `??` syntax was throwing an error.